core-test/src/main/java/org/openstack4j/api/storage/VolumeTests.java

```diff
@@ -63,8 +63,6 @@ public void getVolumeV1() throws Exception {
         assertTrue(getRequest.getPath().matches("/v[12]/\\p{XDigit}*/volumes/8a9287b7-4f4d-4213-8d75-63470f19f27c"));
         
         assertEquals(volume.getId(), "8a9287b7-4f4d-4213-8d75-63470f19f27c");
-        assertEquals(volume.getName(), "vol-test");
```

Comment:
Hi @vinodborole!

As mentioned above - for block storage API v1, volume.getName() and volume.getDescription() now return null values (due to json property change `display_name->name` and `display_description->description`) and these assertions would fail. 

Similar change was applied to CinderVolumeSnapshot.java in #1031. Nevertheless, json property change fixes the response for API v2/v3. Removed the skip from getVolumeV2() which does assert for name and description.